### PR TITLE
Feature/slider states

### PR DESCRIPTION
**Pull request by ingridklepsvik**
Added disabled state to the slider and styling

### CODE REVIEW

File: src/components/general/Slider/Slider.stories.tsx [hunk 1]
```diff
@@ -28,7 +28,12 @@ const SliderPercentageStory = () => {
 
     return (
         <div style={{ width: 400, margin: '0 auto' }}>
-            <Slider value={value} markers={markers} onChange={marker => setValue(marker.value)} />
+            <Slider
+                value={value}
+                disabled
```
Comment (eskil0312) [inline src/components/general/Slider/Slider.stories.tsx]: Do we want the slider in the story to be disabled?

File: src/components/general/Slider/styles.less [hunk 2]
```diff
@@ -107,4 +107,38 @@
...
+    &.isDisabled {
+
+         .dot {
+            border-color: var(--color-black-alt3);
+        }
+
+        &:active {
+                .dot {
+                    border-color: var(--color-black-alt3);
+                    box-shadow: none;
```
Comment (eskil0312) [inline src/components/general/Slider/styles.less]: This section needs some formatting

File: src/components/general/Slider/Marker.tsx [hunk 3]
```diff
@@ -26,14 +27,19 @@ const Marker: React.FC<SliderMarkerProps> = ({ marker, isActive, position, onCli
 
     const markerClassNames = classNames(styles.marker, {
         [styles.isActive]: isActive,
-    })
+        [styles.isDisabled]: disabled,
```
Comment (eskil0312) [inline src/components/general/Slider/Marker.tsx]: Is this used?

Comment (ingridklepsvik) [reply src/components/general/Slider/Marker.tsx]: No. It's now removed

File: src/components/general/Slider/styles.less (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/general/Slider/styles.less]: Formatted now

File: src/components/general/Slider/Slider.stories.tsx (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/general/Slider/Slider.stories.tsx]: We now have two sliders in the story one that is disabled and one active

**Pull request merged**